internal/suite/setup.go

```diff
@@ -143,6 +173,46 @@ func Setup(isUpgrade bool, clusterBuilder cb.ClusterBuilder, clusterReadyFns ...
 	})
 }
 
+func getProviderFromBuilder(clusterBuilder cb.ClusterBuilder) (string, error) {
```

Comment:
Alternative for fetching the provider name, before I had used `suite.Setup(false, "capa", &capa.ChinaBuilder{})` which was not ideal